.drone.yml

```diff
@@ -1,5 +1,11 @@
 ##############################################################################################################
-# Ocaml stuff is based on image tezos/tezos:v8 (if released new one, fell free to upgrade it here)
+# Octez stuff is based on image tezos/tezos:v8 (if released new one, fell free to upgrade it here)
```

Comment:
@adonagy v8 is not actual :), just remove/change it tezos/tezos:v9 or tezos/tezos:vXYZ

Reply:
I did remove the line. On the delphinet tests, we use v8.2 and further down we use v9-release :)